### DISCUSSION TITLE
Mac

### DISCUSSION THREAD

**Discussion by programmermarjann** [community/community#69875] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi
I have a problem with my account, however when I log in it all goes smoothly on my windows laptop but on my Mac it just won't budge,....
I will have to describe the problem cause it won't upload my screenshot either, well it says "one moment please" and that cat is running since last Friday so how can I send the cat to sleep on my Mac please?

**Comment by tarang5757** [comment]
Have you tried restarting your mac rather than putting it to sleep?
Try a different browser and/or try deleting your browser cache

**Comment by dinhtrungndt** [comment]
Report #2
